lib/data-generator.js

```diff
@@ -19,7 +20,25 @@ module.exports = function(runner, done) {
   });
 
   runner.on('test end', function(test) {
-    var error;
+    let error,
+        result = test.ctx.result;
+
+    // go to the main directory to get the assets
+    if (result) {
+      let baseline = result.baseline,
```

Comment:
`let` is ES6 :)

We can start thinking about handling ES6 going forward. We can support only Node 6 which [supports about everything](https://kangax.github.io/compat-table/es6/#node6) or we can transpile with Babel.

For now, it's safe to use the destructuring assignment :)